src/styles/components/header.scss

```diff
@@ -1,2 +1,91 @@
+@import '../mixins';
+
 .header {
+    background: var(--color-background-darker);
+    font-size: 1.6rem;
+    border-bottom: 0.1rem solid #5d5c61;
+    font-weight: var(--weight-semi-bold);
+    
+
+    &__content{
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        padding: 1.5rem 3rem;
+    }
+
+    &__content-item{
+        display: flex;
+        align-items: center;
+        justify-content: center;
+    }
+
+    &__menu-item-container{
+        display: none;
+    }
+
+    &__menu-item{
+        display: inline-block;
+        margin-left: 0.2rem;
+        cursor: pointer;
+        text-decoration: none;    
+
+        &:visited{
+            color: var(--color-text-primary);
+        }
+
+        &:first-child{
+            margin-left: 0;
+        }
+
+        &:hover{
+            color: var(--color-text-secondary);
+        }
+    }
+
+    &__logo-link{
+        cursor: pointer;  
+        display: flex;
+        justify-content: center;
+        padding: 0.5rem 0;
+    }
+
+    &__logo{
+        height: 2rem;
+        pointer-events: none;
+    }
+
+    &__menu-icon{
+        display: flex;
+        justify-content: center;
+        animation: menuItemappear 0.2s linear;
+    }
+
+    @include responsive('tablet'){
+        &__menu-item{
+        font-size: 2rem;
+        margin-left: 3rem;
+        animation: menuItemAppear 0.3s linear;
```

Comment:
since you already have this on mobile you should not need on tablet as well

Reply:
Done